daml-script/dump/src/main/scala/com/daml/script/dump/Config.scala

```diff
@@ -19,6 +23,15 @@ object Config {
   def parse(args: Array[String]): Option[Config] =
     parser.parse(args, Empty)
 
+  private def parseLedgerOffset(s: String): LedgerOffset = LedgerOffset {
+    s match {
+      case s if s == "start" =>
```

Comment:
```suggestion
      case s if s == "begin" =>
```
any reason not to match the name on the API here?